app/src/main/java/com/android/shelfLife/model/foodFacts/FoodFactsViewModel.kt

```diff
@@ -0,0 +1,55 @@
+package com.android.shelfLife.model.foodFacts
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+class FoodFactsViewModel(private val repository: FoodFactsRepository) : ViewModel() {
+
```

Comment:
I wa looking at the ListToDosViewModel, and they use a int{}
the code is :
init {
    repository.init { getToDos() }
  }

I don't know if it is necessary in this project

Reply:
Thanks for the feedback! 
Good point on the `init` block. For `FoodFactsViewModel`, I left it out since the data loads are only triggered by user actions (search by barcode or search by string name). 
We could always add it later if we decide to preload data (automatically search for missing or old information on the current household items), but I dont think it would be necessary for now. Let me know what you think! 
Please accept my PR 🙏🙏🙏🙏🙏🙏